src/extension/debug-page/README.md

```diff
@@ -0,0 +1,13 @@
+# Debug page
+
+## Feature Set
+
+- [x] Backup Database
+- [x] Restore Database
+- [x] Clear Database
+- [ ] Self-tests
+  - [ ] Test R/W Gundb
```

Comment:
Gun db is using local storage not indexed db.

Reply:
me, remove the line

Reply:
Test in Firefox and Chrome